cve_bin_tool/cvedb.py

```diff
@@ -342,11 +343,18 @@ def get_cves(self, vendor, product, version):
 
         # Go through and get all the severities
         if cve_list:
-            query = f'SELECT CVE_number, severity from cve_severity where CVE_number IN ({",".join(["?"]*len(cve_list))}) ORDER BY CVE_number ASC'
+            query = f'SELECT CVE_number, severity, summary from cve_severity where CVE_number IN ({",".join(["?"]*len(cve_list))}) ORDER BY CVE_number ASC'
             cursor.execute(query, cve_list)
             # Everything expects a data structure of cve[number] = severity so you can search through keys
             # and do other easy manipulations
-            return dict(cursor)
+            cves = []
```

Comment:
Can we reuse somehow, the dict we got from scanner module? It would be nice to use same dict instead of creating new one.